Scripts/form_validation.js

```diff
@@ -0,0 +1,35 @@
+const data = document.querySelectorAll('#Form-form input, #Form-form textarea');
+const error = document.querySelectorAll('#Form-form small');
+const form = document.querySelector('#Form-form');
+
+if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
+  data[2].value = 'Sent from my mobile phone';
+}
+
+form.addEventListener('submit', (e) => {
+  for (let i = 0; i < data.length; i += 1) {
+    if (data[i].value === '') {
+      e.preventDefault();
+      data[i].classList.add('error');
+      error[i].textContent = `Fill ${data[i].getAttribute('placeholder')} field first`;
+      error[3].textContent = `Fill ${data[i].getAttribute('placeholder')} field first`;
+      break;
+    }
+    if (data[i].value !== '') {
+      data[i].classList.remove('error');
+      error[i].textContent = '';
+      error[3].textContent = '';
+    }
+  }
+
+  if (data[1].value.toLowerCase() !== data[1].value) {
+    e.preventDefault();
+    data[1].classList.add('error');
+    error[1].textContent = 'Email cant contain upercase';
+    error[3].textContent = 'Email cant contain upercase';
+    e.preventDefault();
```

Comment:
- [ ] Kindly ensure your error message only appears near the submit button only, it appears twice:
![image](https://user-images.githubusercontent.com/61408860/174114263-9fe9c04b-15d5-420c-b59d-790cd0379abe.png)

Reply:
For the code reviewer that just rejected my code without any comment, here is the proof that I fixed the previous reviewer requirements.

![image](https://user-images.githubusercontent.com/85821183/174146032-086861c0-de77-4e73-bda5-832ab1368dd1.png)